writers/postgres/messages.go

```diff
@@ -97,57 +104,94 @@ func (pr postgresRepo) saveSenml(messages interface{}) error {
 	return err
 }
 
-func (pr postgresRepo) saveJSON(messages interface{}) error {
-	msg, ok := messages.(mfjson.Message)
-	if !ok {
-		return errSaveMessage
+func (pr postgresRepo) saveJSON(msgs mfjson.Messages) error {
+	if err := pr.insertJSON(msgs); err != nil {
+		if err == errNoTable {
+			if err := pr.createTable(msgs.Format); err != nil {
```

Comment:
why do we createTable here and in `saveSenml` we dont

Reply:
Because all SenML messages are stored in a single table (created in `init.go`), while JSON messages are stored in separate tables depending on the format. While this does not make too much sense for some writer implementations (such as Postgres - we store payload as a JSON column anyway, so we'll end up with multiple tables with the same schema, but different content of `payload` JSON column), it helps logically organizing messages and easier consuming messages from the DB later.